### PR TITLE
Added nouns and verbs

### PR DESCRIPTION
**Pull request by LockStoch**

### CODE REVIEW

**Review by matkatmusic — CHANGES_REQUESTED** [review]
None of your pseudo-code ends with semicolons, as explained in instruction 4
you didn't write down any plain-english actions.
Please re-read the instructions for this assignment and implement the spec.
- [x] revise

**Comment by LockStoch** [comment]
Cheers MatKat. Those semicolons are definitely a new habit I need to develop!

File: main.cpp [hunk 1]
```diff
@@ -52,85 +52,94 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
...
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  action 1: the fridge cools
+fridge.cool();
+//  action 2:  the fridge deforsts
+firge.defrost();
```
Comment (matkatmusic) [inline main.cpp]: your object doesn't match your noun. 
If this were actual code, you would get a compiler error. 
correct, accurate spelling matters when writing code.   It also helps improve readability.
- [x] revise.

File: main.cpp [hunk 2]
```diff
@@ -52,85 +52,94 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
...
-//  action 3:
- 
 //  8)
+//  Noun: turtle
+//  action 1: the tutrle swims
+turtle.swim();
+//  action 2: the turtle walks
+turtle.walk();
+//  action 3: the turtle hides
+turtle.hide();
```
Comment (matkatmusic) [inline main.cpp]: As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
**REPLACE** the highlighted noun/action as follows:
- [x] come up with a new noun and 3 actions
- [x] pick a noun that needs `camelCase` for the name
- [x] pick actions that need `camelCase` for their name.
**Do not** create an 11th noun.

**Review by matkatmusic — CHANGES_REQUESTED** [review]

File: main.cpp (hunk 1 above)
Comment (LockStoch) [reply main.cpp]: Good pick up.

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**